config/private/plugins.js

```diff
@@ -57,6 +57,9 @@ export default {
         ].filter(x => x != null),
 
         plugins: [
+          // this is used by mobx, and has to list at first
+          // https://mobxjs.github.io/mobx/best/decorators.html
+          'transform-decorators-legacy',
```

Comment:
Sure, I can update them. But what can I do next? Make the change in another new branch(based on your master/next branch) and  send PR again?